zerotwobot/modules/connection.py

```diff
@@ -73,10 +71,9 @@ async def connection_chat(update: Update, context: ContextTypes.DEFAULT_TYPE):
 
     if conn:
         chat = await application.bot.getChat(conn)
-        chat_obj = await application.bot.getChat(conn)
-        chat_name = chat_obj.title
+        chat_name = (await application.bot.getChat(conn)).title
```

Comment:
This is not ideal way of doing, somehow I faced errors too

Reply:
It works...